Sources/MapboxNavigation/NavigationViewportDataSource.swift

```diff
@@ -192,7 +192,8 @@ public class NavigationViewportDataSource: ViewportDataSource {
             let geometryFramingAfterManeuver = followingCameraOptions.geometryFramingAfterManeuver
             let pitchСoefficient = self.pitchСoefficient(routeProgress, currentCoordinate: location.coordinate)
             let pitch = followingCameraOptions.defaultPitch * pitchСoefficient
-            let carPlayCameraPadding = mapView.safeArea + UIEdgeInsets.centerEdgeInsets
+            var carPlayCameraPadding = mapView.safeArea + UIEdgeInsets.centerEdgeInsets
+            carPlayCameraPadding.bottom += 25.0
```

Comment:
Should we move this hardcoded number to a constant? (or maybe even to a config option)

Reply:
`NavigationViewportDataSource` doesn't provide the ability to change specific values of viewport padding for iOS and CarPlay, but completely different padding value can be provided after setting `FollowingCameraOptions.paddingUpdatesAllowed` to false. So I think that it'd be better to leave this part unmodified. I've provided comment, which give more info about the reason why we need to increase bottom padding though.